bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/MieleHandlerFactory.java

```diff
@@ -140,15 +141,15 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
     private ThingUID getBridgeThingUID(ThingTypeUID thingTypeUID, @Nullable ThingUID thingUID,
             Configuration configuration) {
         if (thingUID == null) {
-            String hostID = (String) configuration.get(HOST);
+            String hostID = Objects.requireNonNullElse((String) configuration.get(HOST), "");
```

Comment:
In `bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosHandlerFactory.java` you have a different solution to the very same issue. I would prefer that here as well:
```suggestion
            String hostID = (String) configuration.get(HOST);
            if (hostID == null || hostID.isEmpty()) {
                throw new IllegalArgumentException("The " + HOST + " is not set or is empty in the configuration");
            }
```

Another way to keep it to a minimum would be:
```suggestion
            String hostID = Objects.requireNonNull((String) configuration.get(HOST));
```